simplematrixbotlib/match.py

```diff
@@ -149,11 +155,12 @@ def mention(self):
             Returns True if the message begins with the bot's username, MXID, or pill targeting the MXID, and False otherwise.
         """
 
-        for id in [self._own_disambiguated_name, self._own_display_name, self._own_user_id]:
+        for id in [self._own_disambiguated_name, self._own_display_name, self._own_user_id, self._own_display_name_colon]:
```

Comment:
this will always match _own_display_name and never _own_display_name_colon

Reply:
Perhaps `_own_display_name_colon` could be removed.